.docker/tier1/Dockerfile

```diff
@@ -1,6 +1,17 @@
 FROM dmoj/runtimes-tier1
 
 ARG TAG=master
+
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends unzip && \
+    rm -rf /var/lib/apt/lists/*
```

Comment:
wut is this @hieplpvip

Reply:
Don't install recommended packages and clean up to reduce image size (in theory)